src/main/scala/uk/gov/nationalarchives/tdr/api/auth/AuthorisationTags.scala

```diff
@@ -114,3 +114,16 @@ object ValidateUserOwnsFiles extends AuthorisationTag {
       })
   }
 }
+
+object ValidateHasAntiVirusMetadataAccess extends SyncAuthorisationTag {
+  override def validateSync(ctx: Context[ConsignmentApiContext, _]): BeforeFieldResult[ConsignmentApiContext, Unit] = {
+    val token = ctx.ctx.accessToken
+    val antivirusAccess = token.backendChecksRoles.contains("anti_virus")
```

Comment:
There are going to be a few of these values, so it would be good for them to be constants somewhere so we're not checking against hard-coded strings.

Also, really minor thing, but we're writing "antivirus" as a single word everywhere else, so why is it "anti_virus" in the token?

Reply:
Because thats how  I named it in Keycloak. Have changed it